CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+
+1\. Fork the `pytest-cookies` repo on GitHub.
+
+2\. Clone your fork locally:
+
+``` bash
+$ cd path_for_the_repo
+$ git clone git@github.com:YOUR_NAME/pytest-cookies.git
+```
+
+3\. Assuming you have virtualenv installed (If you have Python3.5 this should already be there), you can create a new environment for your local development by typing:
+
+``` bash
+$ virtualenv pytest-cookies-env
+$ source pytest-cookies-env/bin/activate
+```
+This should change the shell to look something like `(pytest-cookies-env) $``
+
+4\. Create a branch for local development:
+
+``` bash
+$ git checkout -b name-of-your-bugfix-or-feature
+```
+
+Now you can make your changes locally.
+
+5\. When you're done making changes, check that your changes pass flake8. Since, this package contains mostly templates the flake should
+   be run for tests directory:
+
+``` bash
+$ flake8 ./tests
+```
+
+6\. The next step would be to run the test cases. `pytest-cookies` uses tox. Before you run pytest you should tox is installed:
+
+``` bash
+$ pip install tox
+$ tox
+```
+
+7\. Before raising any pull request you should run tox. This will run the tests across different versions of Python:
+
+``` bash
+$ tox
+```
+
+8\. If your contribution is a bug fix or new feature, you may want to add a test to the existing test suite. See section Add a New Test below for details.
+
+9\. Commit your changes and push your branch to GitHub:
+
+``` bash
+$ git add .
+$ git commit -m "Your detailed description of your changes."
+$ git push origin name-of-your-bugfix-or-feature
+```
+
+10\. Submit a pull request through the GitHub website.
+
+Pull Request Guidelines
+-----------------------
+
+Before you submit a pull request, check that it meets these guidelines:
+
+1\. The pull request should include tests.
+
+2\. If the pull request adds functionality, the docs should be updated. Put
+   your new functionality into a function with a docstring, and add the
+   feature to the list in README.rst.
+
+3\. The pull request should work for Python 2.7, 3.3, 3.4 and 3.5, and for PyPy. Check
+   https://travis-ci.org/hackebrot/pytest-cookies/pull_requests
+   and make sure that the tests pass for all supported Python versions.
+
+Add a New Test
+---------------
+When fixing a bug or adding features, it's good practice to add a test to demonstrate your fix or new feature behaves as expected. These tests should focus on one tiny bit of functionality and prove changes are correct.
+
+To write and run your new test, follow these steps:
+
+1\. Add the new test to `tests/test_<file>.py`. Focus your test on the specific bug or a small part of the new feature.
+
+2\. If you have already made changes to the code, stash your changes and confirm all your changes were stashed:
+
+``` bash
+$ git stash
+$ git stash list
+```
+
+3\. Run your test and confirm that your test fails. If your test does not fail, rewrite the test until it fails on the original code:
+
+``` bash
+$ py.test ./tests
```

Comment:
I'd prefer to change this to ``$ tox -e py``, which tells tox to run against the interpreter of the currently active environment.

The difference to running ``$ py.test`` directly is that we check against an sdist distribution and spot potential problems with our ``setup.py``.